config_ens.nml.template

```diff
@@ -1,13 +1,13 @@
 
 &config
  mosaic_file_target_grid="/scratch1/NCEPDEV/global/glopara/fix/orog/20231027/C<RES>/C<RES>_mosaic.nc"
- mosaic_file_input_grid="/scratch1/NCEPDEV/global/glopara/fix/orog/20231027/C384/C384_mosaic.nc"
+ mosaic_file_input_grid="/scratch1/NCEPDEV/global/glopara/fix/orog/20230615/C384/C384_mosaic.nc"
```

Comment:
Does the appropriate file here depend on the version of the downloaded operational restarts?

Reply:
Yes